packages/media-library/src/api/fetch.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import proxy from 'wpcom-proxy-request';
+
+let isAuthorised = false;
+
+const authorise = () => {
+	return new Promise( ( resolve, reject ) => {
+		// TODO: if there is an in-progress authorisation, don't try and authorise, instead wait untill it is resolved
```

Comment:
We use the "authorize" spelling almost all the time 🙂

Reply:
Sorry, hard habit to get out of spelling it correctly 😉🤣